recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -0,0 +1,70 @@
+# Augments upstream linux-firmware with additional and updated images
+# from Raspbian:
+# https://github.com/RPi-Distro/firmware-nonfree
+# https://github.com/RPi-Distro/bluez-firmware
+
+LICENSE_append = "\
+    & Firmware-cypress \
+"
+
+LIC_FILES_CHKSUM_append = "\
+    file://LICENCE.cypress;md5=cbc5f665d04f741f1e006d2096236ba7 \
+"
+NO_GENERIC_LICENSE[Firmware-cypress] = "LICENCE.cypress"
+
+SRC_URI_append = " \
+    git://github.com/RPi-Distro/firmware-nonfree;destsuffix=raspbian-nf;name=raspbian-nf \
+    git://github.com/RPi-Distro/bluez-firmware;destsuffix=raspbian-bluez;name=raspbian-bluez \
+"
+
+SRCREV_raspbian-nf = "86e88fbf0345da49555d0ec34c80b4fbae7d0cd3"
+SRCREV_raspbian-bluez = "e28cd7ee8615de33aa7ec2b41d556af61a4a2707"
+SRCREV_FORMAT = "git"
+
+do_install_append() {
+    install -d ${D}${nonarch_base_libdir}/firmware/brcm/
+
+    # Replace outdated upstream files with fresh from raspbian firmware-nonfree
+    for fw in brcmfmac43430-sdio.bin brcmfmac43455-sdio.bin ; do
```

Comment:
Don't forget the `.txt` NVRAM files :)
Maybe better is to use globs, e.g. `brcmfmac43430-sdio*` and `brcmfmac43455-sdio` ?

Reply:
For reference, see PR #194

Reply:
Those were not in `linux-firmware-raspbian`, and they're not packaged in the OE-core `linux-firmware-bcm43430` so I didn't think they were necessary.  But I confirm that the kernel driver will look for them, so I'll make sure they get packaged (they aren't present in the upstream `linux-firmware`, only in the Raspbian fork).  Thanks.